Ingress/controllers/nginx-third-party/main.go

```diff
@@ -62,14 +62,24 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
+	glog.Info("Checking if DNS is working")
+	ip, err := checkDNS(*defaultSvc)
```

Comment:
can we run this without dns? Or clarify why we need dns in the failure message. In theory we can just serve a default backend in this pod right?